lib/Scheduler/bb_spill.cpp

```diff
@@ -847,21 +854,62 @@ FUNC_RESULT BBWithSpill::Enumerate_(Milliseconds startTime,
 }
 /*****************************************************************************/
 
-InstCount BBWithSpill::UpdtOptmlSched(InstSchedule *crntSched,
-                                      LengthCostEnumerator *) {
+void BBWithSpill::UpdtOptmlSched(InstSchedule *crntSched,
+                                 LengthCostEnumerator *node) {
   InstCount crntCost;
   InstCount crntExecCost;
-
-  //  crntCost = CmputNormCost_(crntSched, CCM_DYNMC, crntExecCost, false);
   crntCost = CmputNormCost_(crntSched, CCM_STTC, crntExecCost, false);
 
-  //#ifdef IS_DEBUG_SOLN_DETAILS_2
-  Logger::Info(
-      "Found a feasible sched. of length %d, spill cost %d and tot cost %d",
-      crntSched->GetCrntLngth(), crntSched->GetSpillCost(), crntCost);
-  //  crntSched->Print(Logger::GetLogStream(), "New Feasible Schedule");
-  //#endif
+  Logger::Event("feasible_sched_found", "length", crntSched->GetCrntLngth(),
+                "spill_cost", crntSched->GetSpillCost(), "cost", crntCost);
+  crntSched->Print(Logger::GetLogStream(), "New Feasible Schedule");
```

Comment:
I think the above feasible sched found event log is good enough. We don't need to print the schedule also.